plugins/appbar/component.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+const React = require('react');
+const Card = require('react-material/components/Card');
+const List = require('react-material/components/List');
+const ListItem = require('react-material/components/ListItem');
+const AppBar = require('react-material/components/AppBar');
+const Overlay = require('react-material/components/Overlay');
+const SideNavigation = require('react-material/components/SideNavigation');
+const Button = require('react-material/components/Button');
+const TextField = require('react-material/components/TextField');
+
+const styles = require('./styles');
+
+const TopBar = React.createClass({
+  getInitialState: function(){
+    return {
+      show: false,
+      chooserType: '',
+      projectName: ''
+    };
+  },
+  showSidebar: function(){
+    this.setState({
+      show: true
+    });
+  },
+  hideSidebar: function(){
+    this.setState({
+      show: false,
+      chooserType: ''
+    });
+  },
+  showNewChooser: function(){
+    this.setState({
+      chooserType: 'new'
+    });
+  },
+  showOpenChooser: function(){
+    this.setState({
+      chooserType: 'open'
+    });
+  },
+  hideChooser: function(){
+    this.setState({
+      chooserType: ''
+    });
+  },
+
```

Comment:
I will update this but I don't agree.  The whitespace separates the methods that do something other than show/hide things